src/services/swap_service.ts

```diff
@@ -629,6 +636,15 @@ export class SwapService {
         };
     }
 
+    public async connectKafkaOrThrow(): Promise<void> {
```

Comment:
Linter will complain

```suggestion
    public async connectKafkaOrThrowAsync(): Promise<void> {
```